src/Register.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import {loginPath} from "./App";
+
+export class Register extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: '',
+            email: '',
+            password: '',
+            repeatPassword: ''
+        }
+    }
+
+    onClick = () => {
+        if (this.validateName() &&
+            this.validateEmail() &&
+            this.validatePassword() &&
+            this.validateRepeatedPassword()
+        ) {
+            alert("Registration successful");
```

Comment:
I agree that it works as validation, but usually, users have a panic attack before they read the text on alert) Usually the text on the form (even red) is much more user-friendly ^^